### DISCUSSION TITLE
Is there any similar project available/known for video transformations?

### DISCUSSION THREAD

**Discussion by hartmut-co-uk** [imgproxy/imgproxy#704] [Q&A]
Hi, I wasn't sure where to put my question - but I was wondering if anyone knows about an existing project similar to imgproxy but for video transformations (like encoding/transcoding)?

Cloudinary example: https://cloudinary.com/documentation/video_manipulation_and_delivery

**Comment by Sumis34** [comment]
I have heard imagekit.io can do something like this stuff.

**Comment by mderazon** [comment]
Cloudflare also does Video transformations:
https://developers.cloudflare.com/stream/transform-videos/

But it would be great if Imgproxy would do that as well
